docs/news.rst

```diff
@@ -90,14 +102,44 @@ New features
 
     (:issue:`456`, :issue:`3477`, :issue:`4467`, :issue:`5627`, :issue:`6729`)
 
+-   Added official support for :ref:`idle <start-requests-idle>` and
+    :ref:`front load <start-requests-front-load>` start request scheduling.
+
+    (:issue:`740`, :issue:`1051`, :issue:`1443`, :issue:`3237`, :issue:`5282`,
+    :issue:`6715`)
+
+-   You can now raise :exc:`~scrapy.exceptions.CloseSpider` from
+    :meth:`~scrapy.Spider.start` and from
+    :meth:`~scrapy.spidermiddlewares.SpiderMiddleware.process_start`.
+
+    (:issue:`3463`, :issue:`4058`, :issue:`6148`, :issue:`6715`, :issue:`6728`)
+
+-   Unlike its precedesors, if :meth:`~scrapy.Spider.start` or
+    :meth:`~scrapy.spidermiddlewares.SpiderMiddleware.process_start` are not
+    asynchronous generators, the crawl starts nonetheless, without start
+    requests.
+
+    This aligns with the behavior of spider callbacks, and allows the crawl to
+    process requests from the scheduler, e.g. when :ref:`resuming a paused job
+    <topics-jobs>`.
+
+    The logged errors have also been improved.
+
+    (:issue:`5426`)
+
 -   :class:`Crawler.signals <scrapy.signalmanager.SignalManager>` has a new
     :meth:`~scrapy.signalmanager.SignalManager.wait_for` method.
 
--   Added a new :signal:`scheduler_empty` signal.
+-   Added new signals: :signal:`spider_start_blocking`,
```

Comment:
Took me some time to come up with `spider_start_blocking`, and still don’t like it. `engine_is_waiting_on_spider_start` or `idle_except_start_requests_are_still_yielding` do not look the part :slightly_smiling_face: